packages/validator/test/unit/utils/interopConfigs.ts

```diff
@@ -383,6 +402,13 @@ export const tekuHoleskyConfig = {
   DOMAIN_AGGREGATE_AND_PROOF: "0x06000000",
   CHURN_LIMIT_QUOTIENT: "65536",
   BLS_WITHDRAWAL_PREFIX: "0x00",
+  MAX_EFFECTIVE_BALANCE_ELECTRA: "2048000000000",
```

Comment:
are these updated based on what Teku actually returns or did we just add this to make tests pass? If those are not actual values from another client the interop config tests are pointless